openstates/md/committees.py

```diff
@@ -14,25 +14,33 @@ def scrape(self, term, chambers):
         doc.make_links_absolute(url)
 
         for a in doc.xpath('//a[contains(@href, "cmtepage")]'):
-            url = a.get('href').replace('stab=01', 'stab=04')
-            chamber = a.xpath('../../..//th/text()')[0]
+            url = a.get('href')
+            chamber_name = a.xpath('../../..//th/text()')[0]
+            if chamber_name == 'Senate Standing':
+                url = url.replace('stab=01', 'stab=04')
+            if chamber_name == 'House Standing':
+                url = url.replace('stab=01', 'stab=04')
             com_name = a.text
             if com_name is None:
                 continue
+            com_name = com_name.replace("Joint ", "")
+            com_name = com_name.replace("Committee on ", "")
+            com_name = com_name.replace(" Committee", "")
```

Comment:
You may want to guarantee that these are where you think they are (ie, make sure that you're not removing "Joint" from the middle of a name, or similar). For example, make the replacements more precise by using regular expressions:
```
import re
…
# The carrot indicates the start of a line in regular expressions
com_name = re.sub("^Joint ", "", com_name)
…
```
or by adding conditionals:
```
if com_name.startswith("Joint "):
    # The last argument means that only the first instance of "Joint " will be replaced
    com_name = com_name.replace("Joint ", "", 1)
```